### DISCUSSION TITLE
Case for adapter v5: How do I open it?

### DISCUSSION THREAD

**Discussion by johnbaersdfsfo** [john30/ebusd#1334] [Q&A]
I ordered the v5 adapter together with the 3D-printed case.
The case came closed and it doesn't open easily. I don't want to apply force and break it.

Can anybody please tell me from which side / edge the case is supposed to be opened? Ideally by providing a picture of the opened case ...

Thanks a lot!
Johannes
![ebus-adapter-case](https://github.com/user-attachments/assets/9d10d96d-a216-407d-a4cc-e8201b73b677)

**Comment by johnbaersdfsfo** [comment]
Anybody please?

**Reply by limad** [reply]
Hello
press between 2 and 4 and pull the upper part
